### ISSUE TITLE
Enzyme translation for mgfplus_style_1 not working

### ISSUE THREAD

**Issue by MKoesters** [uparma/uparma-lib#19]
When setting `enzyme` to nonspecific, the value is not translated properly.

The correct translation for `nonspecific` would be `0`, however the translated_dict looks like this:

```
p = uparma.UParma()
p.translate({'enzyme': 'nonspecific'}, translated_style='msgfplus_style_1', original_style='ursgal_style_1')
>> {
>>      'original_style': 'ursgal_style_1',
>>      'original_key': 'enzyme',
>>      'original_value': 'nonspecific',
>>      'translated_style': 'msgfplus_style_1',
>>      'translated_key': '-e',
>>      'translated_value': 'nonspecific',
>>      'was_translated': True
>> }
```

**Comment by StSchulze** [comment]
Weird, it's in the translations:
```
        [
          "nonspecific",
          "0"
        ],
```
The other translations work for msgfplus (e.g. gluc)?

**Comment by ArtiVlasov** [comment]
This will be the case for all possible value translations where there is no default in the ursgal_style_1 value translation list. Should be solved by PR #20 for this case:

```
'enzyme': {'original_style': 'ursgal_style_1',
  'original_key': 'enzyme',
  'original_value': 'nonspecific',
  'translated_style': 'msgfplus_style_1',
  'translated_key': '-e',
  'translated_value': '0',
  'was_translated': True}}
  ```
